src/dashboard/frontend/src/components/CommandDeck/styles/command-deck.module.css

```
@@ -4694,6 +4694,26 @@
+.forkTargetCwd {
+  margin: 0 20px 12px;
+  padding: 8px 12px;
+  border-radius: 6px;
+  background: color-mix(in srgb, var(--primary) 12%, transparent);
+  color: var(--foreground);
+  font-size: 12px;
+  line-height: 1.45;
+}
+
+.forkTargetCwd code {
+  font-family: "SF Mono", "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)